### PR TITLE
Remove old accountSample items

### PR DESCRIPTION
**Pull request by Isabello**

### CODE REVIEW

File: src/selector/blockchain.mjs [hunk 1]
```diff
@@ -32,6 +32,15 @@ function blockchainExtGetAccountBalanceSelector(state, chainId, address) {
   return -1;
 }
 
+function blockchainExtGetAccountTickersSelector(state, chainId, address) {
+  const account = blockchainExtGetAccountBalanceSelector(state, chainId, address);
+  if (!account) return -1;
+  if (account.hasOwnProperty('balance')) {
```
Comment (Isabello) [inline src/selector/blockchain.mjs]: this can throw if we dont check before.

File: src/action/accountSample.mjs [hunk 2]
```diff
@@ -48,12 +47,14 @@ function performAccountNonceSample(reader, address) {
   };
 }
 
-function performAccountTransactionSample(reader, address) {
-  return (dispatch) => {
+function performAccountNonceSample(address, chainId) {
```
Comment (rudi-cilibrasi) [inline src/action/accountSample.mjs]: why doesn't this use `telescope.chainId`?

Comment (rudi-cilibrasi) [reply src/action/accountSample.mjs]: oh nevermind i see it below! very good

Comment (Isabello) [reply src/action/accountSample.mjs]: this is an adhoc query, not using telescoped address

File: src/action/userProfileManager.mjs [hunk 3]
```diff
@@ -30,9 +30,10 @@ function performInitializeProfile() {
 function performAddPassphraseToKeyring(mnemonic) {
   return (dispatch, getState) => {
     try {
+      console.log(mnemonic);
```
Comment (rudi-cilibrasi) [inline src/action/userProfileManager.mjs]: should remove for safety sake unless you are still using since such sensitive

Comment (Isabello) [reply src/action/userProfileManager.mjs]: missed it, removing

**Pull request merged**